### DISCUSSION TITLE
FLTK with gridster capabilities

### DISCUSSION THREAD

**Discussion by vortex314** [fltk-rs/fltk-rs#1445] [Ideas]
Does FLTK(-RS) has the capabilities to build something like this : https://dsmorse.github.io/gridster.js/demos/resize.html ?
Looking at first glance to the examples of drag and drop , it seems to miss the widget drag capabilities to support this use case ?

**Comment by MoAlyousef** [comment]
fltk-rs supports widget drag/drop capabilities. The linked example should be possible. The Fluid gui designer uses this capability to provide a wsyiwyg experience when designing a ui.

**Comment by vortex314** [comment]
Thanks for the SWIFT answer and confirmation, will try it out further
